console/HAL-1981_Dashboard.adoc

```diff
@@ -0,0 +1,172 @@
+
+=== Cards
+
+==== Product info
+
+This card shows WildFly's static and configuration data. This includes name, version, operation mode, stability level, OS/CPU, and JVM information.
+
+*Sources*
+
+* `:read-resource(attributes-only,include-runtime)`
+* `:product-info()`
+* `/core-service=server-environment:read-resource(include-runtime)`
+* `/host=<domain-controller>/core-service=host-environment:read-resource(include-runtime)`
+
+==== Domain
+
+This card is only visible in domain mode and shows the number and status of domain-related resources like profiles, hosts, server groups, and servers. It shows only the number and status of the resources. All other information is available in the topology view.
+
+*Sources*
+
+* `:read-children-names(child-type=profile)`
+* `:read-children-names(child-type=server-group)`
+* `/host=<host>:read-children-names(child-type=server-config)`
+* `:product-info()`
+
+==== Deployments
+
+This card shows the number and status of the deployments. In domain mode this card shows the deployments per server group.
+
+*Sources*
+
+* `:read-children-resources(child-type=deployment,include-runtime,recursive-depth=2)`
+* `/server-group=*:read-children-resources(child-type=deployment,include-runtime,recursive-depth=2)`
+
+==== Runtime
+
+This card is only available in standalone mode. It shows memory and thread consumption. Therefore, the https://www.patternfly.org/charts/donut-utilization-chart[utilization] chart is used.
+
+*Sources*
+
+* `/core-service=platform-mbean/type=operating-system:read-resource(attributes-only,include-runtime)`
+* `/core-service=platform-mbean/type=runtime:read-resource(attributes-only,include-runtime)`
+* `/core-service=platform-mbean/type=memory:read-resource(attributes-only,include-runtime)`
+* `/core-service=platform-mbean/type=threading:read-resource(attributes-only,include-runtime)`
+
+==== Log
+
+This card shows the last 50 lines from the `server.log` resp. `host-controller.log` file. If the logging subsystem has been configured to use a different name for the log file, the user can specify another name.
+
+*Sources*
+
+* `/subsystem=logging/log-file=server.log:read-log-file(lines=50,tail)`
+
+==== Health
+
+This card is only available in standalone mode. It uses the MicroProfile Health subsystem to show WildFly's healthiness.
+
+* `/subsystem=microprofile-health-smallrye:check-started`
+* `/subsystem=microprofile-health-smallrye:check-ready`
+* `/subsystem=microprofile-health-smallrye:check-live`
+
+If the MicroProfile Health subsystem is not available, this card contains information on how to enable it and a link to https://docs.wildfly.org/32/Admin_Guide.html#MicroProfile_Health_SmallRye.
```

Comment:
If the MicroProfile Health subsystem is not present, you could also first check for the base `health` subsystem too (eg if the server only uses the wildfly-ee galleon pack)